subprojects/docs/src/docs/userguide/managingTransitiveDependencies.adoc

```diff
@@ -140,3 +147,23 @@ A project can decide to disable transitive dependency resolution completely. You
     <sourcefile file="build.gradle" snippet="transitive-per-configuration"/>
 </sample>
 ++++
+
+[[sec:bom_import]]
+=== Importing dependency constraints from Maven BOMs
+
+Gradle provides support for importing https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies[bill of materials (BOM) files], which are effectively `.pom` files that use `<dependencyManagement>` to control the dependency versions of direct and transitive dependencies. The BOM support in Gradle works similar to using `<scope>import</scope>` when depending on a BOM in Maven. In Gradle however, it is done via a regular dependency declaration on the BOM:
+
+++++
+<sample id="importing-dependency-constraints-from-bom" dir="userguide/dependencyManagement/managingTransitiveDependencies/importingDependencyConstraintsFromBOM" title="Depend on a BOM to import it's dependency constraints">
```

Comment:
it's -> its

Reply:
It's start with "Depending..." for the code title.

Reply:
Both fixed.